<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>10000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#Register{
	border-image: url(:/image/images/background1.jpg);
}
</string>
  </property>
  <widget class="QPushButton" name="main_menue_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:default{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-width: 1px;
	border-top-color: rgba(255,150,60,200);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-bottom-color: rgba(200,70,20,200);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:disabled{
	color:rgb(174,167,159);
	border-width: 1px;
	border-radius: 6px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(200, 200, 200, 255), stop:1 rgba(230, 230, 230, 255));
}</string>
   </property>
   <property name="text">
    <string>菜单</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_menue_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:default{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-width: 1px;
	border-top-color: rgba(255,150,60,200);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-bottom-color: rgba(200,70,20,200);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:disabled{
	color:rgb(174,167,159);
	border-width: 1px;
	border-radius: 6px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(200, 200, 200, 255), stop:1 rgba(230, 230, 230, 255));
}</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_menue_btn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>51</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:default{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-width: 1px;
	border-top-color: rgba(255,150,60,200);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-bottom-color: rgba(200,70,20,200);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:disabled{
	color:rgb(174,167,159);
	border-width: 1px;
	border-radius: 6px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(200, 200, 200, 255), stop:1 rgba(230, 230, 230, 255));
}</string>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="about_menue_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>51</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:default{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-width: 1px;
	border-top-color: rgba(255,150,60,200);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-bottom-color: rgba(200,70,20,200);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:disabled{
	color:rgb(174,167,159);
	border-width: 1px;
	border-radius: 6px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(200, 200, 200, 255), stop:1 rgba(230, 230, 230, 255));
}</string>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>321</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(17, 18, 39);

font: 9pt &quot;华文琥珀&quot;;</string>
      </property>
      <property name="text">
       <string>账   号：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border:none;
border-bottom:1px solid lightgray;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(17, 18, 39);

font: 9pt &quot;华文琥珀&quot;;</string>
      </property>
      <property name="text">
       <string>密   码：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border:none;
border-bottom:1px solid lightgray;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(17, 18, 39);

font: 9pt &quot;华文琥珀&quot;;</string>
      </property>
      <property name="text">
       <string>确认密码：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border:none;
border-bottom:1px solid lightgray;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-bottom-color: rgb(150,150,150);
	border-right-color: rgb(165,165,165);
	border-left-color: rgb(165,165,165);
	border-top-color: rgb(180,180,180);
	border-style: solid;
	padding: 4px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:hover{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:default{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius:6px;
	border-top-color: rgb(255,150,60);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 255));
	border-bottom-color: rgb(200,70,20);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:pressed{
	color:rgb(17,17,17);
	border-width: 1px;
	border-radius: 6px;
	border-width: 1px;
	border-top-color: rgba(255,150,60,200);
	border-right-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-left-color:  qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(200, 70, 20, 255), stop:1 rgba(255,150,60, 200));
	border-bottom-color: rgba(200,70,20,200);
	border-style: solid;
	padding: 2px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:1, stop:0 rgba(220, 220, 220, 255), stop:1 rgba(255, 255, 255, 255));
}
QPushButton:disabled{
	color:rgb(174,167,159);
	border-width: 1px;
	border-radius: 6px;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:1, x2:0.5, y2:0, stop:0 rgba(200, 200, 200, 255), stop:1 rgba(230, 230, 230, 255));
}</string>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>exit_menue_btn</zorder>
  <zorder>reset_menue_btn</zorder>
  <zorder>about_menue_btn</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>main_menue_btn</zorder>
 </widget>
 <resources>
  <include location="../images_res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>main_menue_btn</sender>
   <signal>clicked(bool)</signal>
   <receiver>Register</receiver>
   <slot>show_hide_menue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>-78</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_menue_btn</sender>
   <signal>clicked()</signal>
   <receiver>Register</receiver>
   <slot>exit_ui()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>-47</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>about_menue_btn</sender>
   <signal>clicked()</signal>
   <receiver>Register</receiver>
   <slot>about_cnn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>-88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reset_menue_btn</sender>
   <signal>clicked()</signal>
   <receiver>Register</receiver>
   <slot>resnet_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>-52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Register</receiver>
   <slot>check_register()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>590</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>show_hide_menue()</slot>
  <slot>exit_ui()</slot>
  <slot>about_cnn()</slot>
  <slot>resnet_data()</slot>
  <slot>check_register()</slot>
 </slots>
</ui>
